Process.c

```diff
@@ -374,17 +374,15 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
       xSnprintf(buffer, n, "%*d ", Process_pidDigits, this->tgid);
       break;
    case TPGID: xSnprintf(buffer, n, "%*d ", Process_pidDigits, this->tpgid); break;
-   case TTY_NR: {
-      unsigned int major = major(this->tty_nr);
-      unsigned int minor = minor(this->tty_nr);
-      if (major == 0 && minor == 0) {
+   case TTY:
+      if (!this->tty_name) {
          attr = CRT_colors[PROCESS_SHADOW];
-         xSnprintf(buffer, n, "(none)   ");
+         xSnprintf(buffer, n, "(no tty) ");
```

Comment:
Shouldn't this be `N/A` to be consistent with other columns when data is unavailable?

Reply:
In my understanding 'N/A' means some error happened or htop does not have sufficient privileges to gather the data.
But (most) processes just do not have a controlling terminal, so `(no tty)` is a very common case.